iOS/Core/PixelEvent.swift

```diff
@@ -2058,20 +2056,26 @@ extension Pixel.Event {
         case .debugBookmarksMigratedMoreThanOnce: return "m_debug_bookmarks_migrated-more-than-once"
             
         case .configurationFetchInfo: return "m_d_cfgfetch"
+        case .couldNotLoadConfiguration(let configuration, let target):
+            switch target {
+            case .app: return "m_debug_\(configuration)_load_failed".lowercased()
+            case .vpn: return "m_debug_\(configuration)_load_failed_\(target.rawValue)".lowercased()
+            }
```

Comment:
For peace of mind I'd consider adding tests just to check the returned name is correct based on params.